src/components/pages/maintenance/summary/summary.js

```diff
@@ -6,20 +6,65 @@ import { NavLink } from 'react-router-dom';
 
 import { Notifications } from './notifications';
 import { Jobs } from './jobs';
-import { PageContent, ContextMenu } from 'components/shared';
+import { PageContent, ContextMenu, RefreshBar, Svg } from 'components/shared';
+import { svgs } from 'utilities';
 
-export const Summary = ({ match: { params: { path } } }) => [
-  <ContextMenu key="context-menu"></ContextMenu>,
-  <PageContent className="maintenance-container" key="page-content">
-    <div className="header">Maintenance</div>
+import './summary.css';
+
+const renderCount = (value) => typeof value === 'undefined' ? '--' : value;
```

Comment:
Yay! a helper for this format.  But since similar format is needed in other files, perhaps a more shared location would be useful.